### PR TITLE
fix(replay): Fix Replay DOM Tab loading status and new indicator

### PR DESCRIPTION
**Pull request by ryan953**
SCR-20230820-mvol
SCR-20230820-mvqc

### CODE REVIEW

File: static/app/views/replays/detail/filterLoadingIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import {space} from 'sentry/styles/space';
+
+interface Props {
+  children: ReactNode;
+  isLoading: boolean;
+}
+
+export default function FilterLoadingIndicator({children, isLoading}: Props) {
+  return (
+    <Wrapper style={{}}>
```
Comment (billyvg) [inline static/app/views/replays/detail/filterLoadingIndicator.tsx:16]: Empty style?

**Review by billyvg — COMMENTED** [review]
This might look better with indicator "inside" of the input so that there's no jumpy dom when it gets loaded, but no strong opinion

**Comment by ryan953** [comment]
> This might look better with indicator "inside" of the input so that there's no jumpy dom when it gets loaded, but no strong opinion

On the new Trace/Perf tab there's no input box (yet) so it's also outside. This PR is a bug-fix, but that new tab was really the motivator.
The forthcoming a11y tab will also need this indicator as the data streams in, so maybe it's better as a question for design to follow up on.

File: static/app/views/replays/detail/filterLoadingIndicator.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/replays/detail/filterLoadingIndicator.tsx:16]:
```suggestion
    <Wrapper>
```

**Review by billyvg — APPROVED** [review]

**Pull request merged**